models/PartnerBillingParameters.v1.json

```diff
@@ -0,0 +1,19 @@
+{
+  "title": "PartnerBillingParameters",
+  "description": "Paramètres permettant de générer une facture à destination d'un partenaire.",
+  "type": "object",
+  "properties": {
+    "funding_results": {
+      "$ref": "./FundingResults.v1.json"
+    },
+    "case": {
+      "$ref": "./Case.v1.json"
+    },
+    "partner_from": {
```

Comment:
Is this the payload for the POST call to create a bill?
If that's the case shouldn't we have IDs instead of full objects?

Reply:
Yes, I imagined that we will have 2 endpoints: the standalone bill generator, and a one taking only IDs, like for the simulation, no ?